apps/web/views.py

```diff
@@ -29,7 +29,7 @@ def home(request, template_name="index.html"):
     return render(request, template_name)
 
 
-def page_not_found(request):
+def page_not_found(request, exception):
```

Comment:
Why are we passing `exception` as a parameter when we are not using it?

Reply:
@RishabhJain2018 please refer [this](https://docs.djangoproject.com/en/3.1/ref/views/). In django 2.0 `page_not_found` function has exception as mandatory param.